modules/portmap.nix

```diff
@@ -0,0 +1,10 @@
+{}: {
+    domain = "fklub.dk";
+    #domain = "localhost"; # Comment the line above an uncomment this to run locally
+
+    ports = {
+        stregsystemet = "8080";
```

Comment:
This seems like it belongs more naturally in a config option defined in `modules/stregsystemet.nix` and then specified in a module in `system`, given the split of `modules` being for implementation, and `system` being for configuration.

Reply:
can you just pass `port` to an entry in the imports list? my thought was to have one file for raw networking options that are used by multiple services

The idea here was to set the port for stregsystemet in one place where its changed in all other modules to avoid having to change it in too many places at once

Reply:
ah i think i see what you mean, i have an idea of what to change in modules to make both of us happy

Reply:
ah another problem is if we want to test the system in a vm, its quite tedious to change options in nginx.nix, stregsystemet.nix and others just to change the host domain to localhost, which portmap.nix (bad name i know) did reasonably well

Reply:
I think we should have a slightly different configuration specifically for VM testing, so we can do `nixos-rebuild build-vm .#fklub-test` for that and then `nixos-rebuild switch .#fklub` for deploying. That way we save the trouble of manually editing anything. That's a separate PR though.